specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabricManagedClusters/stable/2021-05-01/managedcluster.json

```diff
@@ -1168,7 +1178,11 @@
         "Deleting",
         "Deleted",
         "Other"
-      ]
+      ],
+      "x-ms-enum": {
+        "name": "ManagedResourceProvisioningState",
+        "modelAsString": true
```

Comment:
changing to model enums as string again as suggested by Jeffrey Richter on the intake: https://msazure.visualstudio.com/One/_workitems/edit/9744666/?triage=true